src/experiment.rs

```diff
@@ -25,10 +27,14 @@ pub struct Experiment {
     add_pulls: Option<f32>,
 }
 
+impl Experiment {
+}
+
```

Comment:
?

Reply:
oopsie.  cruft left after letting the IDE create the method.